app/src/main/kotlin/com/adesso/movee/scene/movie/MovieViewModel.kt

```diff
@@ -53,7 +55,9 @@ class MovieViewModel @Inject constructor(
             val nowPlayingMoviesResult = fetchNowPlayingMoviesUseCase.run(UseCase.None)
 
             onUIThread {
-                nowPlayingMoviesResult.either(::handleFailure, ::postNowPlayingMovieList)
+                nowPlayingMoviesResult
+                    .onSuccess { postNowPlayingMovieList(it) }
```

Comment:
it can be passed as a function reference, like `onSuccess(::postNowPlayingMovieList)`

Reply:
You are right, that is better. Updated the PR.